packages/ui/src/organisms/authentication/other-sign-options.tsx/index.tsx

```diff
@@ -41,10 +43,15 @@ export const AuthOtherSignOptions = ({
   return (
     <div>
       <IconCmpArrow
-        className="absolute w-6 transition-opacity cursor-pointer hover:opacity-50 shrink-0"
+        className="absolute w-6 transition-opacity cursor-pointer hover:opacity-50 shrink-0 top-5 left-5"
         onClick={onBack}
       />
-      <AuthAppMeta applicationURL={appMeta} />
+      <AuthAppMeta
+        applicationURL={appMeta}
+        withLogo={!isIdentityKit}
+        title={isIdentityKit ? "Sign in" : undefined}
+        subTitle={isIdentityKit ? "to continue to" : undefined}
```

Comment:
lets pass prop like title and subTitle, not isIdentitykit

Reply:
here are 3 values depend on identitykit: withLogo, title, subtitle.
Do you mean to pass these values as props from coordinator ?

Reply:
exactly

Reply:
DOne